### ISSUE TITLE
Demo is unstyled

### ISSUE THREAD

**Issue by Strangehill** [scotch-io/scotch-io.github.io#2]
There's no style on the [demo](https://scotch-io.github.io/)

**Comment by whatnickcodes** [comment]
Right now, HTTP only. http://scotch-io.github.io/

**Comment by Strangehill** [comment]
It must be just me, but I'm redirected to the https site.

**Comment by whatnickcodes** [comment]
Okay - I'll look into. Or, update the code to be http/https agnostic

**Comment by Strangehill** [comment]
Yep. I've got the [https everywhere](https://www.eff.org/https-everywhere) chrome extension!

**Comment by brainhell** [comment]
I've forked your repository, but the page is unstyled please see https://brainhell.github.io/blogger/.
Any help about this will be appreciated.

Thanks in advance.......
